lib/perl/Genome/Model/ClinSeq/Command/UpdateAnalysis.pm

```diff
@@ -1141,7 +1175,7 @@ sub check_ref_align_models{
     next unless ($self->determine_model_data_type('-model'=>$model) eq $data_type);
 
     #If the desired $data_type is exome.  Check that the TRSN and ROI have been set correctly, exclude models that are not
-    next unless ($self->check_model_trsn_and_roi('-model'=>$model, '-data_type'=>$data_type));
+#    next unless ($self->check_model_trsn_and_roi('-model'=>$model, '-data_type'=>$data_type));
```

Comment:
This was not intentional, I had commented this out as a test and forgot to return it to its original state. Thanks for the catch.